docker/Dockerfile.hdfs

```diff
@@ -1,10 +1,13 @@
-FROM ubuntu:22.04 as base
+FROM ubuntu:22.04 AS base
 
 ENV LANG en_US.utf8
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install make build-essential cmake protobuf-compiler curl pkg-config bash lld maven unzip wget openjdk-11-jdk libsasl2-dev
```

Comment:
`DEBIAN_FRONTEND=noninteractive` I remeber we need this to change the cmd to noninteractive mod, or when running the dockerfile, it will stucked. But I am not sure wether this related to the system version.

Reply:
So let's just add it back.

Reply:
One more thing need to pay attention:
`DEBIAN_FRONTEND=noninteractive apt-get -y install make build-essential cmake protobuf-compiler curl pkg-config bash lld maven unzip wget openjdk-11-jdk libsasl2-dev`
I see you change this into two lines. I am not sure `DEBIAN_FRONTEND=noninteractive` is specific for which package installation. Please make sure they will not stuck when running, thanks.